src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/prison/PrisonEventListenerIntTest.kt

```diff
@@ -49,9 +51,10 @@ class PrisonEventListenerIntTest : MessagingMultiNodeTestBase() {
     val email = randomEmail()
     val cro = randomCro()
     val postcode = randomPostcode()
-    val firstName = randomFirstName()
-    val lastName = randomLastName()
-    stubPrisonResponse(prisonNumber = prisonNumber, pnc = pnc, email = email, cro = cro, postcode = postcode, firstName = firstName, lastName = lastName)
+    val prefix = randomFirstName()
+    val personDateOfBirth = randomDateOfBirth()
+
+    stubPrisonResponse(prisonNumber = prisonNumber, pnc = pnc, email = email, cro = cro, postcode = postcode, prefix = prefix, dateOfBirth = personDateOfBirth)
```

Comment:
Might as well as add `ApiResponseSetup ` as the argument instead of increasingly add arguments to match it. with a default object?

Reply:
good idea